.internal-ci/docker/Dockerfile.bootstrap-tools

```diff
@@ -16,17 +16,17 @@ ARG DEBIAN_FRONTEND=noninteractive
 
 # Install updates
 RUN apt-get update \
- && apt-get install -y \
-        ca-certificates \
-        gettext \
-        libssl1.1 \
-        jq \
-        vim \
-        python3-pip \
-        zstd \
-        openssl \
- && rm -rf /var/cache/apt \
- && rm -rf /var/lib/apt/lists/*
+  && apt-get install -y \
```

Comment:
The indents here were purposeful to show that these were options under the `apt-get install` command. It would be nice to have them back.